tests/ci/tee_popen.py

```diff
@@ -11,23 +16,42 @@
 # stdout.
 class TeePopen:
     # pylint: disable=W0102
-    def __init__(self, command, log_file, env=os.environ.copy()):
+    def __init__(self, command, log_file, env=os.environ.copy(), timeout=None):
         self.command = command
         self.log_file = log_file
         self.env = env
         self.process = None
+        self.timeout = timeout
+
+    def _check_timeout(self):
+        stopwatch = Stopwatch()
+        while stopwatch.duration_seconds < self.timeout:
+            sleep(1)
```

Comment:
Maybe just call sleep for (`self.timeout`)?

Reply:
![yes](http://i.imgur.com/eYR6luF.gif)